examples/vulkan/Vulkan.cpp

```diff
@@ -17,7 +17,8 @@
 #include <limits>
 #include <vector>
 
-
+// Disable clang-tidy for this file
+// NOLINTBEGIN
```

Comment:
```suggestion
// NOLINTBEGIN(modernize-use-nullptr)
```
As written, _all_ clang-tidy checks get silence for this block.

Reply:
I don't know why, but clang-tidy was changing non-pointer 0-values to `nullptr` in this file, so I disabled it after then initial conversion

Reply:
I played around with this yesterday and it appears that on Visual Studio 2019, the Vulkan API itself is expecting an int instead of a pointer. Perhaps we use `{}` for these arguments so that we can avoid disabling this check but still make it work on both builds.

Reply:
I did that for a couple, I'll try using `{}` in the constructor initialization list, which is where most of the problematic ones were

Reply:
You can replace `(0)` in the initializer list with `()` to default initialize the members. That will give you `0` when the members are integers and `nullptr` when the members are pointers.